roles/variables/vars/main.yml

```diff
@@ -238,6 +260,29 @@ kafka_controller_properties:
     enabled: "{{kafka_controller_telemetry_ansible_labels_enabled}}"
     properties:
       confluent.telemetry.labels.confluent.ansible.playbooks.version: "{{confluent_ansible_branch}}"
+  audit_logs_destination:
+    enabled: "{{audit_logs_destination_enabled and rbac_enabled}}"
```

Comment:
Not sure why rbac is required for audit logs. Lets discuss this

Reply:
I'm not sure about that either, but I copied the condition from [here](https://github.com/confluentinc/cp-ansible/blob/7.3.3-post/roles/variables/vars/main.yml#L393)

Reply:
will address in separate PR